docs/public-networks/reference/api/index.md

```diff
@@ -846,6 +846,73 @@ None
           ]
         }
         ```
+### `debug_getRawBlock`
+
+Returns the selected block as RLP-encoded.  
+
+#### Parameters
+
+#### Returns
+
+`result`: *object* - [block object](objects.md#block-object)
+
+!!! example
+
+    === "curl HTTP request"
+
+        ```bash
+        curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawBlock","params":["0x32026E"],"id":1}' http://127.0.0.1:8545
```

Comment:
> "params":["0x32026E"]

What is this parameter?

Reply:
its a random sepolia block hex i chose - can be 0x1 but wont correlate with the results I provide